src/main/paradox/docs/api/assets/acls-add.sh

```diff
@@ -0,0 +1 @@
+curl -XPUT -H "Content-Type: application/json" "https://nexus.example.com/v1/acls/org1" -d '{"acl": [{"permissions": ["read"], "identity": {"@type": "Group", "realm": "realm"} }, {"permissions": ["read", "write"], "identity": {"@type": "Group", "realm": "realm", "group": "some-group"} }, {"permissions": ["acls/read", "acls/write"], "identity": {"@type": "User", "realm": "realm", "subject": "alice"} } ] }'
```

Comment:
could you please format this to make it easier to read? using the `'` char allows the use of `LF`.